src/lib.rs

```diff
@@ -1,3 +1,6 @@
+//! This implements a handling of a pass directory compatible with https://www.passwordstore.org/ .
+//! The encryption is handled by GPGme and the git integration is with libgit2.
+
```

Comment:
Should this change be a part of this PR?

Reply:
minor thing, two pr's was too much work